app/assets/stylesheets/govuk_publishing_components/components/_layout-super-navigation-header.scss

```diff
@@ -700,6 +700,15 @@ $search-icon-height: 20px;
   .gem-c-layout-super-navigation-header__header-logo {
     padding: 10px 0 !important; // stylelint-disable-line declaration-no-important
   }
+
+  // Hide menu links, buttons and dropdown
+  .gem-c-layout-super-navigation-header__navigation-item-link,
+  .gem-c-layout-super-navigation-header__navigation-item,
+  .gem-c-layout-super-navigation-header__search-item-link,
+  .gem-c-layout-super-navigation-header__search-item,
+  .gem-c-layout-super-navigation-header__navigation-dropdown-menu {
+    display: none;
+  }
```

Comment:
Two possible ways to shorten this block of styles:

```scss
.gem-c-layout-super-navigation-header__content {
  > :nth-child(n+3){
    display: none;
  }
}
  ```
Or i'm not sure you need to include a selector for `gem-c-layout-super-navigation-header__navigation-item-link` or `gem-c-layout-super-navigation-header__search-item-link` since they're nested in the `-item` styles:
```scss
  // Hide menu links, buttons and dropdown
  // .gem-c-layout-super-navigation-header__navigation-item-link,
  .gem-c-layout-super-navigation-header__navigation-item,
  // .gem-c-layout-super-navigation-header__search-item-link,
  .gem-c-layout-super-navigation-header__search-item,
  .gem-c-layout-super-navigation-header__navigation-dropdown-menu {
    display: none;
  }
```

Reply:
Good catch, I've gone with removing the redundant selectors for now, looks much cleaner:

```
  // Hide menu links, buttons and dropdown
  .gem-c-layout-super-navigation-header__navigation-item,
  .gem-c-layout-super-navigation-header__search-item,
  .gem-c-layout-super-navigation-header__navigation-dropdown-menu {
    display: none;
  }
```